lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -702,6 +702,8 @@ sub _applicable_upgrade_tests {
                   LedgerSMB::Upgrade_Tests->get_tests;
 }
 
+my @checks;
```

Comment:
Our requests shouldn't depend on global state (I'm going through a lot of work to *eliminate* global state, such as App_State). Is there a reason you declare the checks as global?